NanoServerPackage/NanoServerPackage.psm1

```diff
@@ -198,34 +187,30 @@ function Save-NanoServerPackage
 
         if($Path)
         {
+            $ExceptionObject = $Path
             $destinationPath = Resolve-PathHelper -Path $Path `
-                                                    -CallerPSCmdlet $PSCmdlet | Microsoft.PowerShell.Utility\Select-Object -First 1
-
-            if(-not $destinationPath -or -not (Microsoft.PowerShell.Management\Test-path $destinationPath))
-            {
-                $errorMessage = ("Cannot find the path '{0}' because it does not exist" -f $Path)
-                ThrowError  -ExceptionName "System.ArgumentException" `
-                            -ExceptionMessage $errorMessage `
-                            -ErrorId "PathNotFound" `
-                            -CallerPSCmdlet $PSCmdlet `
-                            -ExceptionObject $Path `
-                            -ErrorCategory InvalidArgument
-            }
+                                                  -CallerPSCmdlet $PSCmdlet | Microsoft.PowerShell.Utility\Select-Object -First 1
         }
         else
         {
+            $ExceptionObject = $LiteralPath
```

Comment:
No, as this is mandatory in LiteralPath related parameter sets.

---
In reply to: [82316147](https://github.com/OneGet/NanoServerPackage/pull/6#discussion_r82316147) [](ancestors = 82316147)